draft-ietf-cbor-time-tag.md

```diff
@@ -247,18 +249,19 @@ Intents might include information about time zones, daylight savings
 times, preferred calendar representations, etc.
 
 Semantics not covered by this document can be added by registering
-additional map keys for the map inside the tag, the specification for
-which is referenced by the registry entry ({{map-key-registry}}, {{time-format}}).
-For example, map keys could be registered for:
-
-* Direct representation of natural platform time formats.  Some
-  platforms use epoch-based time formats that require some computation
-  to convert them into the representations allowed by tag 1; these
-  computations can also lose precision and cause ambiguities.
-  (The present specification does not take a position on whether tag 1
-  can be "fixed" to include, e.g., Decimal or BigFloat representations.
-  It does define how to use these representations with the extended
-  time format.)
+additional map keys for the map that is the content of the tag (see
+`etime-detailed` in {{tag-1001}}),
+the specification for
+which is referenced by the registry entry (see {{time-format}}).
+
+For example, map keys could be registered for direct representations
+of natural platform time formats.  Some platforms use epoch-based
+time formats that require some computation to convert them into the
```

Comment:
Non-blocking nit:
```suggestion
of natural platform time formats.  Some platforms use epoch-based
time formats that require computation to convert them into the
```

Reply:
I actually like the qualification as to what amount of computation is needed (i.e., not much).